src/lib.rs

```diff
@@ -343,6 +343,13 @@ pub struct TrashItemMetadata {
     pub size: TrashItemSize,
 }
 
+#[derive(Debug, Clone, Copy, Default)]
+pub enum RestoreMode {
+    Force,
```

Comment:
The variants could probably be less abstract by spelling out more closely what they do.